data/deep missions.txt

```diff
@@ -1122,32 +1131,32 @@ mission "Deep: Project Hawking: Carbuncle"
+			`	"Anyhow, because of my past work, the Deep approached me to work on this project. The pay was better than what I was getting before so I accepted it. It was a pleasant surprise to meet my partners as well. Hannah graduated in the same class as me, double majoring in quantum mechanics and ancient history, of all things."`
 			choice
 				`	"What is it that you need from Carbuncle Station that is so secret?"`
-				`	"What about Isaac and Laura?"`
+				`	"What about Pierre and Laura?"`
 					goto partners
 			
 			`	"Like I said already, the project is classified. It's not something I would understand, though. It's in Laura's field of expertise. You see, she majored in-"`
 				goto end
 			
 			label partners
 			branch electron
-				has "event: deep sky tech available"
-			`	"Isaac is an older guy. He has been working on laser weaponry for the better part of three decades now. Before this he was working on a seperate project, but the Deep asked that he take time away from that to work on Project Hawking while they try to find a replacement for him, since the project needed someone with better knowledge in laser weaponry than the rest of us.`
+				has "event: navy using mark ii ships"
+			`	"Pierre is an older guy, he's been working on laser weaponry for the better part of three decades now. Before this he was working on a separate project, but the Deep asked that he take time away from that to work on Project Hawking while they try to find a replacement for him, since the project needed someone with better knowledge in laser weaponry than the rest of us.`
 				goto laura
 			
 			label electron
-			`	"Isaac is an older guy. He has been working on laser weaponry for the better part of three decades now. If you've ever tried out an electron beam, you have him to thank for that. And a few others, of course, but one of his findings is really what brought the whole project together.`
+			`	"Pierre is an older guy. He has been working on laser weaponry for the better part of three decades now. If you've ever seen an electron beam in action, you have him to thank for that. And a few others, of course, but one of his findings is really what brought the whole project together.`
 			
 			label laura
 			`	"Laura, on the other hand-"`
 			
 			label end
-			`	Before Garrison can finish speaking, the other three scientists return, helping each other carry a sealed metal box. "We're just going to sit this in the back, if you don't mind," Isaac says. The scientists make themselves cozy in the bunk rooms and get ready to launch.`
-				accept
+			`	Before Garrison can finish speaking, the other three scientists return, helping each other carry a sealed metal box. "We're just going to set this in the back, if you don't mind," Pierre says. The scientists make themselves cozy in the bunk rooms and get ready to depart.`
+				launch
```

Comment:
This doesn't seem necessary to me, especially since the other two missions don't cause you to launch.

Reply:
I think it adds to the mystery of Carbuncle

Reply:
Alright, makes sense; the player is going to need to depart and refuel somewhere else anyways.
`"Now get out," the workers say as you board your ship.`